flowsa/common.py

```diff
@@ -36,10 +40,17 @@
 crosswalkpath = datapath + 'activitytosectormapping/'
 flowbysectormethodpath = datapath + 'flowbysectormethods/'
 flowbysectoractivitysetspath = datapath + 'flowbysectoractivitysets/'
-fbaoutputpath = outputpath + 'FlowByActivity/'
 fbsoutputpath = outputpath + 'FlowBySector/'
 
-local_storage_path = appdirs.user_data_dir()
+paths = Paths
+paths.local_path = os.path.realpath(paths.local_path + "/flowsa")
+
+pkg = pkg_resources.get_distribution("flowsa")
```

Comment:
This is writing out for me as 0.0.0.1 (which means it appears to have an extra 0?)

Reply:
fxn output for me is 0.0.1, could we preface with a 'v' to clarify it is the version?

Reply:
For me it was also just 0.0.1 -- I'm interested in seeing that. Yes I  will add the 'v' which is consistent with the schema. That is done in esupy

Reply:
In reviewing this step for lciaformatter, I'm wondering whether how a package gets installed could be affecting this. I found I needed to update my egg because back when I originally installed, setup.py indicated a much earlier version. Do you all do an editable install, and if so, do you manually update your egg? (i used python setup.py bdist_egg to do so).

I believe this explains my original version being 0.0.0.1.
Since Catherine recently updated the version number in flowsa, I wonder if in pycharm for you all your egg will update automatically or if it will still mark output files as v0.0.1
@WesIngwersen @catherinebirney

Reply:
I'm having trouble with this step too. setup.py is updated to reflect version v0.0.2, but any FBA or FBS output parquet still reflects version v0.0.1, which is the version indicated in my egg

Reply:
I wonder if we could set a global variable in each package, which would get used within setup.py, and could also be passed to the meta definition for esupy. That way you change it in one spot and wouldn't have to worry about re-creating the egg each time. I suppose, fortunately, this is only an issue for developers and not users.